### PR TITLE
Support Exposing new Metric properties using EventSource

### LINKED ISSUE #87150: Support Meter and Instrument Tags in MetricsEventSource
We have recently introduced the ability to attach tags to Meters and Instruments during their creation. This ongoing issue focuses on ensuring that the tags associated with Meters and Instruments are included in the source events published by MetricsEventSource, particularly in out-of-process scenarios.

https://github.com/dotnet/runtime/issues/77514#issuecomment-1553414738

### PR DESCRIPTION
**Pull request by tarekgh**
Fixes https://github.com/dotnet/runtime/issues/87150

### CODE REVIEW

File: src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregatorStore.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ namespace System.Diagnostics.Metrics
     //   it may also not be a win to further pessimize the slow-path (JIT compilation is expensive) to squeeze yet more cycles out of
     //   the fast path.
     //   - Allocations per lookup: Any lookup of 3 or fewer labels on the above fast path is allocation free. We have separate
-    //   dictionaries dependending on the number of labels in the list and the dictionary keys are structures representing fixed size
+    //   dictionaries depending on the number of labels in the list and the dictionary keys are structures representing fixed size
```
Comment (noahfalk) [inline src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/AggregatorStore.cs:37]: Haha, that was some interesting spelling I came up with ;p

**Review by noahfalk — APPROVED** [review]
LGTM 👍

**Comment by tarekgh** [comment]
Opened https://github.com/dotnet/runtime/issues/90439 for the unrelated failure.

**Pull request merged**